src/legacy/server/kbn_server.d.ts

```diff
@@ -55,7 +55,7 @@ declare module 'hapi' {
 type KbnMixinFunc = (kbnServer: KbnServer, server: Server, config: any) => Promise<any> | void;
 
 export default class KbnServer {
-  public readonly core: any;
+  public readonly newPlatform: any;
```

Comment:
Can we expose the types for this?

Reply:
I didn't do because we are dealing with it via untyped Hapi sever. Although, I just realized that `x-pack` plugins could benefit from it in the future, so will fix.

Reply:
@epixa done